### PR TITLE
azure: kustomization.yaml udpate path to ssh key

### PR DESCRIPTION
**Pull request by knrt10**
Fixes: #552
Signed-off-by: Kautilya Tripathi <ktripathi@microsoft.com>

### CODE REVIEW

File: install/overlays/azure/kustomization.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ secretGenerator:
 - name: ssh-key-secret
   namespace: confidential-containers-system
   files: # key generation example: ssh-keygen -f ./id_rsa -N "" && sudo cat id_rsa.pub >> /root/.ssh/authorized_keys
```
Comment (snir911) [inline install/overlays/azure/kustomization.yaml:40]: also this comment is not needed as it used to authorize the public key on the libvirt machine

**Comment by snir911** [comment]
Can you verify it works? as we define it with file permissions needed for the private key IDK if it has any affect when it's public key (probably it doesn't matter)
Also it may require additional changes to work with the controller, can be done later if needed i guess

File: install/overlays/azure/kustomization.yaml (hunk 1 above)
Comment (surajssd) [reply install/overlays/azure/kustomization.yaml:40]: The part after `&&` unnecessary!

**Review by snir911 — APPROVED** [review]
LGTM , thanks!

**Review by surajssd — APPROVED** [review]

**Pull request merged**